CONTRIBUTING.md

```diff
@@ -11,17 +11,26 @@ If you are not working on a windows OS, .Net Frameworks are not supported but yo
 
 Before running tests please ensure that the environment variables are set up for the test project.
 
+```sh
```

Comment:
I think all these variables are only needed when running the integration tests? Maybe we could make that clearer in the readme like

### Unit tests

Run `make test-with-docker`

### Integration tests

export the stuff and run make ...